ignite/cmd/generate_vuex.go

```diff
@@ -38,7 +38,7 @@ func generateVuexHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	s.Stop()
-	fmt.Println("⛏️  Generated vuex stores.")
+	fmt.Println("⛏️  Generated Typescript Client and Vuex stores")
```

Comment:
Can we also print the path?

Please check `network chain prepare` cmd for the format.

Reply:
See previous [discussion](https://github.com/ignite/cli/pull/2143#discussion_r967357109)